### DISCUSSION TITLE
2.6.15 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3469] [讨论]
· 修改 MCBBS 的特别鸣谢描述，删除论坛入口，给加了回去
· 更新 2 月赞助者列表，感谢大家的支持！
· 修复了以下 3 个 Bug（包含 1 个严重 Bug）
　　[严重] 无法安装 Forge（#3462，@Frenqy，by @bangbang93）
　　部分高校源不允许使用浏览器 UA 直接下载（#3452）
　　帮助 [使用 GitHub Desktop 获取帮助库] 无法打开（#3456，LTCatt/PCL2Help#354，@SYBass，by @JingHai-Lingyun）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.15'>2.6.15</a>.</em>

**Comment by hyyyp** [comment]
包内的最新版zip  版本号依然是 2.6.14 是版本号忘记改 还是 压缩错文件？

**Reply by allMagicNB** [reply]
你再试试？或者说你用的 CDN 还没同步

**Reply by hyyyp** [reply]
![23232](https://github.com/Hex-Dragon/PCL2/assets/6407577/df6426cc-f486-4982-bd1a-05beebd2d2de)

**Reply by allMagicNB** [reply]
> ![23232](https://github.com/Hex-Dragon/PCL2/assets/6407577/df6426cc-f486-4982-bd1a-05beebd2d2de)
> 

自己删了重新下载再看看
不要下载`2.6.15`Tag 中的版本

**Reply by hyyyp** [reply]
重启电脑可以了  怪了  明明退出启动器了 并且覆盖了 最新版本 再运行的  咋会显示老版本号呢
